dracut-sshd.spec

```diff
@@ -33,12 +33,19 @@ cp -r 46sshd %{buildroot}/usr/lib/dracut/modules.d/
 %files
 /usr/lib/dracut/modules.d/46sshd/module-setup.sh
 /usr/lib/dracut/modules.d/46sshd/sshd.service
+/usr/lib/dracut/modules.d/46sshd/motd
+/usr/lib/dracut/modules.d/46sshd/profile
```

Comment:
Ok, strictly speaking this goes beyond Silverblue compatibility and would be content for another pull-request.

But I don't want to be more catholic than the pope so you can leave it in.

Reply:
I'm not sure if your COPR build worked fine after the `motd` and `profile` patch, but mine failed on  these missing from the spec file. I specifically added this patch to be able to test my RPM (and keep my systems updated until you accept my PR). Thanks for the exception, though, I'll pay attention in case of future patches to make sure I don't mix the functionality.